crates/solidity/testing/snapshots/cst_output/VersionPragma/multiple_exact_versions/generated/0.4.11.yml

```diff
@@ -8,18 +8,16 @@ Errors: []
 Tree:
   - VersionPragma (Rule): # 0..20 "solidity 0.7.0 0.8.0"
       - Solidity (Token): "solidity" # 0..8
-      - Group: # 9..20 "0.7.0 0.8.0"
+      - _ANON (Rule): # 9..20 "0.7.0 0.8.0"
           - VersionPragmaSpecifier (Rule): # 9..14 "0.7.0"
-              - Group: # 9..14 "0.7.0"
-                  - VersionPragmaValue (Token): "0" # 9..10
-                  - Period (Token): "." # 10..11
-                  - VersionPragmaValue (Token): "7" # 11..12
-                  - Period (Token): "." # 12..13
-                  - VersionPragmaValue (Token): "0" # 13..14
+              - VersionPragmaValue (Token): "0" # 9..10
```

Comment:
The CST structure changed here. Under `VersionPragmaSpecifier` there should be two nodes, an `optional(VersionPragmaSpecifier)`, and a `separatedBy(".", VersionPragmaValue)`. In sibling tests, this is produced correctly, but here, when there is no operator, the second child is promoted up.

Am I reading this correctly? CST shape should not change depending on whether one child exists or not. Otherwise, it would be much harder to reuse or visit the tree.

Reply:
Not sure why was this resolved. I think that is still the case here.

Reply:
done

Reply:
Not sure why you think is is unresolved. The current output looks ok to me:

```
Tree:
  - VersionPragma (Rule): # 0..20 "solidity 0.7.0 0.8.0"
      - Solidity (Token): "solidity" # 0..8
      - _REPEATED (Rule): # 8..20 " 0.7.0 0.8.0"
          - VersionPragmaSpecifier (Rule): # 8..14 " 0.7.0"
              - _SEPARATEDBY (Rule): # 8..14 " 0.7.0"
                  - VersionPragmaValue (Token): "0" # 9..10
                  - Period (Token): "." # 10..11
                  - VersionPragmaValue (Token): "7" # 11..12
                  - Period (Token): "." # 12..13
                  - VersionPragmaValue (Token): "0" # 13..14
          - VersionPragmaSpecifier (Rule): # 14..20 " 0.8.0"
              - _SEPARATEDBY (Rule): # 14..20 " 0.8.0"
                  - VersionPragmaValue (Token): "0" # 15..16
                  - Period (Token): "." # 16..17
                  - VersionPragmaValue (Token): "8" # 17..18
                  - Period (Token): "." # 18..19
                  - VersionPragmaValue (Token): "0" # 19..20
  ```

Reply:
Resolved by your reply on the other thread. Thanks!